transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -448,6 +448,9 @@ def backward(
                 mu,
                 rsigma,
             ) = restore_from_saved(ctx.tensor_objects, saved_tensors)
+            # Delete the references to tensor objects once they've been consumed
+            # by the `restor_from_saved` method to construct back the actual tensors.
```

Comment:
nit: restor**e**_from_saved, same for the subsequent comments.

Reply:
ys, I should also fix the  ky in my kyboard